mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
IMO introducing different speedups for each queue skews hurts the simplicity of our gameplay rules, which is a downside for casual players.  Would it be possible to apply these same rates to all of the queues?